CTS POD/AppDelegate.swift

```diff
@@ -14,6 +14,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         // Override point for customization after application launch.
+        LocationManager.sharedInstance.checkLocationPermission()
```

Comment:
Location permission will check after customer entry screen 
on Dashboard screen.

Reply:
yes I know, but let it be we can put there once we will implement that functionality